web/src/selectors/groups.js

```diff
@@ -0,0 +1,32 @@
+export default (
```

Comment:
if we're not going to name this function can you add a comment explaining what it does?